app/src/interfaces/useSearchApi.interface.ts

```diff
@@ -43,8 +43,22 @@ export interface IGetOccurrenceData {
   geometry: Feature | null;
 }
 
-export type ISearchResponse<T = unknown, S = unknown> = Array<{
+export type IElasticsearchResponse<T = unknown, S = unknown> = Array<{
   id: string;
   fields: T;
-  source?: S;
+  source: S;
 }>;
+
+export interface IKeywordSearchResult {
+  project: [
```

Comment:
Just checking: can the project field have multiple items in the array, or just 1?

The way its defined now is that project is an array with exactly 1 item.

`project: [type]` vs `project: type[]`

Reply:
Ahh good catch!